src/utils/transaction.ts

```diff
@@ -28,7 +28,7 @@ export async function submitTransaction(
 
 		showLoadingDialog('Submitting transaction');
 
-		const result = await transaction.signAndSubmit(signer);
+		const result = await transaction.signAndSubmit(signer, { at: 'best' });
```

Comment:
This seems to still return `TxFinalizedPayload` so it's probably waiting for finalization anyways, we might need to go the observable route using `signSubmitAndWatch` to be able to wait only for inclusion.

Reply:
Like in the new version?